packages/zoe/test/unitTests/contracts/fakeVatAdmin.js

```diff
@@ -3,23 +3,40 @@ import { makePromiseKit } from '@agoric/promise-kit';
 
 import { evalContractBundle } from '../../../src/contractFacet/evalContractCode';
 
-export default harden({
-  createVat: bundle => {
-    return harden({
-      root: E(evalContractBundle(bundle)).buildRootObject(),
-      adminNode: {
-        done: () => {
-          const kit = makePromiseKit();
-          // Don't trigger Node.js's UnhandledPromiseRejectionWarning
-          kit.promise.catch(_ => {});
-          return kit.promise;
+function makeFakeVatAdmin(testContextSetter = undefined, makeRemote = x => x) {
+  // This is explicitly intended to be mutable so that
+  // test-only state can be provided from contracts
+  // to their tests.
+  const admin = harden({
+    createVat: bundle => {
+      return harden({
+        root: makeRemote(
```

Comment:
this `makeRemote` doesn't appear to be used, is there some future patch that needs it, or was it just for debugging?

Reply:
It is used in the dapp-token-economy.  Once this is landed, I'll ty it in these tests (but expect to encounter issues).